### PR TITLE
Add help link to discord

### LINKED ISSUE #1414: Add help button with link to discord technical help channel

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #1414

### CODE REVIEW

**Comment by Brutus5000** [comment]
Maybe add help in forum as well?

**Comment by 1-alex98** [comment]
Discord is cooler who reads the forums anyway XD 😛

**Comment by 1-alex98** [comment]
image

**Comment by 1-alex98** [comment]
@PhilipJFryFAF so after that you can take away your irc topics

**Comment by Katharsas** [comment]
- Can your either order or structure these links in a way that makes sense? Seems to be random order right now.
- I would remove facebook, its even more dead than the devblog (which could also be removed as long as nobody is willing to write the blog). 
- The button is called "Help on Discord", which doesn't make much sense, "Help" would be better.

File: src/main/java/com/faforever/client/main/MainController.java [hunk 1]
```diff
@@ -120,6 +120,7 @@
   private final GamePathHandler gamePathHandler;
   private final PlatformService platformService;
   private final VaultFileSystemLocationChecker vaultFileSystemLocationChecker;
+  private final ClientProperties clientProperties;
```
Comment (Katharsas) [inline src/main/java/com/faforever/client/main/MainController.java]: Unused

File: src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -700,3 +701,12 @@ mapDir.changed.noneAscii=We detected you have a username with special characters
...
 mapDir.changed.oneDrive=We detected you use OneDrive for your documents and moved your vault directory to fallback location.
 settings.use.secondaryVaultLocation=Use Fallback vault location. You want to use this if map/mod downloading has issues and the game does not find maps/mods.
+help.title=Links Overview
+linksTecHelpDiscord=Technical help on discord
+linksTecHelpForum=Technical help on forum
+linksRules=Rules
+linksModerationReport=Moderation Report Form
+linksDiscord=Discord
+linksFacebook=Facebook
+linksDevNews=Dev News
```
Comment (Katharsas) [inline src/main/resources/i18n/messages.properties]: I would call this Dev Blog and not Dev News since its not really News.

**Review by Katharsas — CHANGES_REQUESTED** [review]

**Comment by 1-alex98** [comment]
Well moderators want these links. Those are the links from irc channel topic, I gues you will need to talk to them about this

File: src/main/java/com/faforever/client/main/LinksAndHelpController.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import com.faforever.client.theme.UiService;
+import javafx.scene.Node;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.VBox;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/main/LinksAndHelpController.java:15]: Why is this prototype scope? It never changes.

Comment (1-alex98) [reply src/main/java/com/faforever/client/main/LinksAndHelpController.java:15]: Controllers always have SCOPE_PROTOTYPE

**Review by Brutus5000 — DISMISSED** [review]
Maybe check for katharsa's comment. I don't care that much about the naming.

File: src/main/resources/theme/chat/channel_tab.fxml [hunk 4]
```diff
@@ -13,7 +13,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.text.TextFlow?>
 <?import javafx.scene.web.WebView?>
-<Tab xmlns:fx="http://javafx.com/fxml/1" fx:id="channelTabRoot" xmlns="http://javafx.com/javafx/11.0.1"
+<Tab xmlns:fx="http://javafx.com/fxml/1" fx:id="channelTabRoot" xmlns="http://javafx.com/javafx/10.0.2"
```
Comment (1-alex98) [inline src/main/resources/theme/chat/channel_tab.fxml:16]: Fixes #1422

File: .idea/compiler.xml [hunk 5]
```diff
@@ -6,6 +6,10 @@
       <profile default="true" name="Default" enabled="true" />
     </annotationProcessing>
     <bytecodeTargetLevel>
+      <module name="downlords-faf-client.main" target="1.10" />
+      <module name="downlords-faf-client.test" target="1.10" />
+      <module name="downlords-faf-client.webview-patch.main" target="1.10" />
+      <module name="downlords-faf-client.webview-patch.test" target="1.10" />
       <module name="downlords-faf-client_main" target="1.10" />
```
Comment (Katharsas) [inline .idea/compiler.xml:13]: Maybe add this file to gitignore?

Comment (1-alex98) [reply .idea/compiler.xml:13]: not really they should be commited

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**